assembly/compliance-tool/src/main/java/org/eclipse/sw360/antenna/frontend/compliancetool/sw360/status/InfoRequest.java

```diff
@@ -0,0 +1,55 @@
+package org.eclipse.sw360.antenna.frontend.compliancetool.sw360.status;
```

Comment:
Should the package name be something like "report" or "reporter"? "status" is pretty generic.